charts/super-agent/values.yaml

```diff
@@ -94,71 +94,76 @@ super-agent-deployment:
   # @default -- `false`
   nrStaging:
 
-  # -- (bool) Changes default endpoint to point to EU backend.
-  # @default -- `false`
-  euEndpoints:
-
   fedramp:
     # -- (bool) TODO: Enables FedRAMP. Can be configured also with `global.fedramp.enabled`
     # @default -- `false`
     enabled:
 
-  # -- (bool) TODO: Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
+  # -- (bool) Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
   # @default -- `false`
   verboseLog:
 
   # -- Enable the cleanup of super-agent managed resources when the chart is uninstalled.
-  # If disabled, agents and / or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
+  # If disabled, agents and/or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
   cleanupManagedResources: true
 
   config:
-    auth:
-      enabled: false
-      organizationId:
-      userApiKey:
-      secret:
-        create: true
-
     # -- Configuration for the Super Agent.
     # @default -- See `values.yaml`
     superAgent:
       # -- Set if the configMap is going to be created by this chart or the user will provide its own.
       create: true
-      # -- Here you can set New Relic' Super Agent configuration.
-      # @default -- See `values.yaml` for examples
-      content:
-        agents:
-          open-telemetry:
-            agent_type: newrelic/io.opentelemetry.collector:0.2.0
-
-        # opamp:
-        #   # TODO The endpoint should be set automatically based on the licenseKey and on the nrStaging option if opamp.enable=true
-        #   endpoint: https://opamp.service.newrelic.com/v1/opamp
-        #   endpoint: https://opamp.service.eu.newrelic.com/v1/opamp
```

Comment:
I no longer see opamp defined in the values.yaml. How does it work? is it enabled by default?

Reply:
I think that we are enabling opamp if auth is enabled. Should we review UX with Jose? I find it confusing now

Reply:
Endpoint for OpAmp is calculated the same way as `tokenRenewal` and `systemIdentityRegistration`.

I'll ping José so he knows.

Reply:
After an offline conversation, you were right. I changed here: [#8984a70](https://github.com/newrelic/helm-charts/pull/1466/commits/8984a702336fc8bd11bcdc37d28ed0f210b9f0e5)

Reply:
I am not sure if we should error out if opamp=true while auth is not configured 🤔

Reply:
what do you think?

Reply:
From my point of view, auth is coupled to opamp. There is no way to use opamp without authenticating so the only toggle that is being used is the one at `.config.opamp.enabled`.